### PR TITLE
Add scale up/down controls on deployments, replica sets, and replication controllers

### PR DESCRIPTION
**Pull request by paulbellamy**
@tomwilkie, ptal

Was debating adding a test for the `CaptureResource` method, as it's a bit involved.

### CODE REVIEW

File: probe/kubernetes/reporter.go [hunk 1]
```diff
@@ -326,3 +341,18 @@ func (r *Reporter) podTopology(services []Service, replicaSets []ReplicaSet) (re
 	})
 	return pods, err
 }
+
+func addScalingControls(t report.Topology) {
```
Comment (tomwilkie) [inline probe/kubernetes/reporter.go]: Can we do something similar to https://github.com/weaveworks/scope/blob/master/probe/docker/reporter.go#L53?

**Comment by tomwilkie** [comment]
Looks good, but I think we might be able to make this more generic by using the Meta type in CaptureResource.  Will think on it.

**Comment by paulbellamy** [comment]
> using the Meta type

Ah, maybe.. The tricky bit, at the moment, is mapping some "ReplicaSet" nodes back to replication controllers. Maybe that can be handled more gracefully, which would simplify CaptureResource.

**Comment by tomwilkie** [comment]
> Will think on it.

I didn't think of a better way.   LGTM

**Pull request merged**